meerk40t/newly/gui/newlycontroller.py

```diff
@@ -149,7 +149,8 @@ def on_button_start_connection(self, event):  # wxGlade: Controller.<event_handl
             self.context("usb_connect\n")
 
     def pane_show(self):
-        name = self.service.label
+        name = self.service.label.replace(" ", "-")
+        name = name.replace("/", "-")
         self.context.channel(f"{name}/usb").watch(self.update_text)
```

Comment:
This is strictly speaking allowed but a lot harder to listen to the channel since you'd need to quote the damn thing, so this is probably better.